UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/appsettings.Development.json

```diff
@@ -13,4 +13,13 @@
         "PageSize": "100",
         "Start": "0"
     }
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
```

Comment:
Please remove duplicate entry for Logging config.

Reply:
done required changes